src/amolMule/array2/ArrayOprns.java

```diff
@@ -0,0 +1,124 @@
+package amolMule.array2;
+
+/*
+ Write a method to return sum of all not prime numbers from given array.
+input : {11,13,17,22,44}
+output : 66
+Hint : (22+44 = 66)
+*/
+public class ArrayOprns {
+
+	int getSumNP(int[] arr) {
+		int s = 0;
+		for (int i = 0; i < arr.length; i++) {
+
+		}
+
+		return s;
+	}
+
+	boolean checkPrime(int n) {
+		boolean b = true;
+		int flag = 0;
+		for (int i = 2; i < n; i++) {
+			if (n % i == 0) {
+				flag = 1;
+				b = false;
+				break;
+		
+			}
+
+		}
+		return b;
+	}
+	
+	int sumNP(int []arr) {
+		int sum=0;
+		for(int i=0;i<arr.length;i++) {
+		boolean b1=checkPrime(arr[i]);
+		if(b1==false) {
```

Comment:
you can put checkPrime(arr[i]) method inside if brackets as it require boolean value & that method returns boolean value